schedule/yast/installer_extended/installer_extended_aarch64.yaml

```diff
@@ -18,9 +18,8 @@ schedule:
     - installation/isosize
     - installation/data_integrity
     - installation/bootloader_start
-  product_selection:
+  early_keyboard_tests:
```

Comment:
this is confusing for several reasons:
- Inventing keys that do not correspond with the screen, if you would like to go in that way we wouldn't use only one part of the screen, but the whole: `language_keyboard_license`.
- This extended scenario is specific, only happens once, it shouldn't belong to a default yaml file.
- The word 'test' should not be used for testing most of the time to name anything, it is redundant most of the time.

For keeping current design and still have clarity (referring to your intent to remove unnecesary product selection for this case) we could do this:
```
product_selecition: []
license_agreement:
  - installation/language_keyboard/switch_keyboard_layout
  - installation/licensing/verify_license_translations
```